aptos-move/aptos-vm-types/src/change_set.rs

```diff
@@ -37,6 +37,20 @@ pub struct GroupWrite {
     pub inner_ops: HashMap<StructTag, WriteOp>,
 }
 
+impl GroupWrite {
+    /// Utility method that extracts the serialized group size from metadata_op.
+    pub fn encoded_group_size(&self) -> VMResult<u64> {
+        self.metadata_op
+            .bytes()
+            .map_or(Ok(0), |b| bcs::from_bytes::<u64>(b))
```

Comment:
it seems simpler to use `to_le_bytes`/`from_le_bytes` then bcs to encode the size

Reply:
kinda disagree - it is easy to have one side do one thing, another side another, and have discrepancies. if we are always using bcs - we are always using bcs

Reply:
yes, I can see both sides, I just don't like the long unnecessary error handling, probably for cases like this we should just use unreachable

Reply:
made a constructor to ensure invariants, used bcs but with expect